js/ethfinex2.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const bitfinex2 = require ('./bitfinex2.js');
+
+//  ---------------------------------------------------------------------------
+
+module.exports = class ethfinex2 extends bitfinex2 {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'ethfinex2',
+            'name': 'Ethfinex v2',
+            'countries': [ 'VG' ],
+            'version': 'v2',
+            'rateLimit': 1500,
+            // new metainfo interface
+            'certified': false,
+            'has': {
+                'CORS': false,
+                'createDepositAddress': true,
```

Comment:
The issue with it is that we can't set `createDepositAddress: true`, here, because it will require implementing that method for Bitfinex2 first. In fact, the Bitfinex2 itself is currently broken for the same reason – we have to update the metainfo there and we have to add those missing methods first to make it work. I suggest that we exclude the ethfinex2.js file from this PR for now, until we have it thoroughly implemented. Basically, the current Ethfinex implementation requires a complete overhaul, and I'd rather do that in the `ethfinex.js` file without adding `ethfinex2.js`. Does that sound ok to you? If you agree, I will then merge it without `ethfinex2.js` for now. If not, let me know your thoughts, plz.

Reply:
@Matoran alternatively, I can merge it without `ethfinex2.js` right now (the yobit fix) and we would return to ethfinex2 later? How does that sound to you?

Reply:
Yes, it will be perfect. This pull request should have been split into 3 :P.